formSkeleton/inputFields/input-fields.ttl

```diff
@@ -1197,7 +1197,7 @@ fields:f70a9c35-a58f-4943-bdfc-7b9a761ed04c a form:Field ;
 fields:d0f11e8a-8a57-4309-91ff-e92e40d27525 a form:Field ;
     mu:uuid "d0f11e8a-8a57-4309-91ff-e92e40d27525" ;
     sh:name "Nieuw aangelegde fietsinfrastructuur op eigen grondgebied." ;
-    form:help """ Maak een optelsom van het aantal meter niew aangelegde of structureel verbeterde fietsinfrastructuur. Een tweerichtingsfietspad (aan één kant van een weg), wordt in de LEKP-doelstelling meegeteld als een lengte in afstand. """ ;
+    form:help """ Maak een optelsom van het aantal meter niew aangelegde of structureel verbeterde fietsinfrastructuur. Een tweerichtingsfietspad aan één zijde van de weg telt als één afstand in de LEKP-doelstelling, waarbij de lengte van A naar B wordt genoteerd. Bij een fietspad aan beide zijden kan het als eenrichtingsfietspad gelden in twee richtingen: van A naar B en van B naar A. """ ;
```

Comment:
Noticed a small typo. Other than that it's not very clear what the expected texts are supposed to be, so I'll leave it at this 😄.

```suggestion
    form:help """ Maak een optelsom van het aantal meter nieuw aangelegde of structureel verbeterde fietsinfrastructuur. Een tweerichtingsfietspad aan één zijde van de weg telt als één afstand in de LEKP-doelstelling, waarbij de lengte van A naar B wordt genoteerd. Bij een fietspad aan beide zijden kan het als eenrichtingsfietspad gelden in twee richtingen: van A naar B en van B naar A. """ ;
```